### PR TITLE
Blockcrafter step-by-step page

### PR DESCRIPTION
**Pull request by ligos**
Changes:

* New blockcrafter page with excessive hand holding.
* Blockcrafter page is what gets linked to in other parts of the docs.
* Bumped the version numbers so docs reflect mapcrafter 3.0 & Minecraft 1.13.

Looks like my last rebasing worked to some extent. But my separate branch didn't remove all the old commits though, so actual changes are all from 10 Nov.

One outstanding item is stepping people through docker blockcrafter usage. I don't have the right environment to test it, so I'll have to leave it to you.

### CODE REVIEW

File: docs/blockcrafter.rst [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+`Blockcrafter Github page <https://github.com/mapcrafter/blockcrafter>`_.
+
+.. note::
+    Mapcrafter already contains block sprites for ``texture_size`` 12 and 16, 
+    all ``render_views`` (isometric, topdown, side), and all ``rotations`` 
+    (top-left, top-right, bottom-right, bottom-left).
+
+    You only need Blockcrafter:
+        * To use a ``texture_size`` other than 12 or 16.
+        * To render a map which uses a custom Minecraft resource pack.
```
Comment (m0r13) [inline docs/blockcrafter.rst:21]: Maybe a note that this includes Minecraft mods.

File: docs/blockcrafter.rst [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+
+Installing and Using Blockcrafter
+=================================
+
+Blockcrafter is a `Python 3 <https://www.python.org/>`_ app, which uses 
+`VisPy <http://vispy.org>`_ to render images. To use it, you'll need:
```
Comment (m0r13) [inline docs/blockcrafter.rst:28]: "... vispy as OpenGL rendering framework"

**Comment by m0r13** [comment]
Hey, I'm sorry I keep you waiting again. There are so many projects and things I want to do, there is hardly enough time for everything. I'll try to review it this week.

**Comment by ligos** [comment]
> so many projects and things I want to do

I know the feeling. I'm in no rush.

**Comment by elamperti** [comment]
Hi @ligos, fantastic work on the documentation! Thanks for that :blush: 

  * I'm wondering if we should keep `MCVERSION` in 1.13 considering #348 would make Mapcrafter compatible with 1.15 as well (though I don't see a change for this file in that branch). I'm not familiar with the impact of this file.
  * I see that you are modifying 3 PNG files in `docs/img/`, and the diff shows that the change is only in file size (but not in the image content or its resolution), so: is there a reason to update those images? In the three cases the files are increasing in size.

Other than that it looks really good! :slightly_smiling_face:

**Comment by ligos** [comment]
Hi @elamperti,

Wow. It's been a long time since I made this pull request!!

Re: `MCVERSION` - if you're going to merge support for 1.15, then no point reverting to 1.13. I think this appears in the `--version` command line option. Don't remember if it had any other effect. (Remembering I'm a C# developer, my C++ remains quite rusty).

Re: PNGs - It's possible I re-did them to be 32bit PNGs. I remember I started at 8bit but @m0r13 asked me to create higher quality versions. It's equally possible I mucked something up. Been too long. Hmmm... they seem to be pixel perfect identical. I wonder if it was a case sensitivity issue (I was working on Windows).

Are you able to cherry pick / edit my pull request? It's not easy for me to resubmit at the moment.

Murray

**Comment by elamperti** [comment]
Thanks for the context and data on `MCVERSION`! I see I can edit the PR and even remove the PNGs so don't worry, I can take them out before merging if needed. Thanks again!

**Comment by m0r13** [comment]
I fixed some small things... `s/mapcrafter-export/blockcrafter-export/` in some places :D 

Docker documentation is still a TODO, but we can do this in a separate issue.

**Comment by elamperti** [comment]
Added #280 to track Docker documentation. If further modifications are required, please create a new issue or PR. Thank you!

I'll be reverting the PNG changes (dc81a0a) in a separate PR.

**Pull request merged**